spacewalk/package/spacewalk.spec

```diff
@@ -118,7 +118,7 @@ Requires:       postgresql13-contrib
 Conflicts:      postgresql-implementation >= 14
 Conflicts:      postgresql-contrib-implementation >= 14
 %endif # if sle_version >= 150400
-%else # not suse_version
+%else # not suse_version or opensuse
```

Comment:
EL is using 13.5. Does the logic force a different version? (if it does, then it's not working :) )

Reply:
the code forced to use a postgres version between 12 and 14, I just changed it from 13 to 15 (15 is not released yet).

Reply:
ACK. Thanks.

@sbluhm well, on paper EL should use the same version Uyuni uses for openSUSE.

Until now it was OK, because we were based on Leap 15.3 and PostgreSQL 13. But Uyuni 2022.06 will change to Leap 15.4 and PostgreSQL 14.

EL8 should be working, as I don't recall we are using anything specific that only works for PostgreSQL 14. But now that we have it on Uyuni for Leap, that could change.